### PR TITLE
Remove support for iOS 10

### PR DESCRIPTION
**Pull request by kyleve**
This removes support for iOS 10 since SPOS has now dropped it. Note that I have _not_ changed CI in this PR; since it's significantly re-worked here: https://github.com/square/Blueprint/pull/152, once this lands, we can then update to only run tests on 11 through 14.

### CODE REVIEW

File: SampleApp/Sources/PostsViewController.swift [hunk 1]
```diff
@@ -43,14 +43,6 @@ final class PostsViewController: UIViewController {
         blueprintView.element = element
     }
 
-    private var viewSafeAreaInsets: UIEdgeInsets {
```
Comment (kyleve) [inline SampleApp/Sources/PostsViewController.swift:46]: This probably broke the tutorial build. Also a good time to make sure these build as part of the SampleApp!

File: SampleApp/SampleApp.xcodeproj/xcshareddata/xcschemes/SampleApp.xcscheme [hunk 2]
```diff
@@ -20,6 +20,62 @@
                ReferencedContainer = "container:SampleApp.xcodeproj">
             </BuildableReference>
          </BuildActionEntry>
+         <BuildActionEntry
```
Comment (kyleve) [inline SampleApp/SampleApp.xcodeproj/xcshareddata/xcschemes/SampleApp.xcscheme:23]: This makes the tutorials build as part of the Cocoapods build; so we can't accidentally break them.

File: SampleApp/Sources/PostsViewController.swift (hunk 1 above)
Comment (kyleve) [reply SampleApp/Sources/PostsViewController.swift:46]: Nope, it didnt; wasnt used! But added tutorials as part of SampleApp build.

**Comment by kylebshr** [comment]
This is breaking since CI currently runs on iOS 10 - how about we merge #152 first, and drop the iOS 10 device from this PR?

**Comment by kyleve** [comment]
Yep, sounds good

**Comment by kyleve** [comment]
(I actually intended that, but forgot to note it here; which is why I didnt touch the .travis.yml file)

**Review by watt — APPROVED** [review]

**Comment by kylebshr** [comment]
It's merged! You can rebase and remove the iOS 10 sim from the matrix

**Comment by kylebshr** [comment]
Went ahead and updated it again since I merged the yml change and it was conflicting

**Pull request merged**